rest-service/manager_rest/storage/management_models.py

```diff
@@ -174,12 +174,8 @@ def tenants(cls):
         ]
         constraint = db.PrimaryKeyConstraint(
             'user_id', 'tenant_id', name='users_tenants_pkey')
-        secondary_table = db.Table(
-            'users_tenants',
-            constraint,
-            *columns
-
-        )
+        args = columns + [constraint]
+        secondary_table = db.Table('users_tenants', *args)
```

Comment:
Maybe we can do something like this?
```py
secondary_table = db.Table('users_tenants', *columns, constraint=constraint)
```
That way we won't need the awkward `args`